src/bundler.js

```diff
@@ -577,7 +588,15 @@ function getOrCreateStylusCss(options, stylusText, stylusPath, cssPath, cb /*cb(
 
 function getOrCreateMinCss(options, css, cssPath, minCssPath, cb /*cb(minCss)*/) {
     compileAsync(options, "minifying", function (css, cssPath, cb) {
-            cb(new CleanCss(options).minify(css));
+            new CleanCss({
+                advanced: false,
+                restructuring: false
+            }).minify(css, function(err, minCss) {
+                if (err) {
+                    throw err;
+                }
+                cb(minCss.styles);
+            });

```

Comment:
The syntax of `CleanCss` changed a bit, and it's now async. Also, we had been passing in `options` which are our Bundler options, not anything having to do with CSS minification. Now we're passing in options that disable a few of the newer (and riskier) features added to `CleanCss` so that the output is more consistent with our current version.